common/ECSResourcesCheck/src/main/java/Comparer.java

```diff
@@ -0,0 +1,70 @@
+import org.json.JSONObject;
+public class Comparer {
+
+    public void compare(JSONObject jsonBase, String cluster,ECSResources ecs) {
```

Comment:
Rename "cluster" to "clusterName" to make it clear.

```suggestion
    public void compare(JSONObject jsonBase, String clusterName, ECSResources ecs) {
```